src/docdb/getCosmosClient.ts

```diff
@@ -4,8 +4,14 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { CosmosClient } from "@azure/cosmos";
+import * as https from "https";
+import * as vscode from 'vscode';
 import { appendExtensionUserAgent } from "vscode-azureextensionui";
+import { ext } from "../extensionVariables";
+
+export function getCosmosClient(endpoint: string, key: string, isEmulator: boolean | undefined): CosmosClient {
+
```

Comment:
Can you confirm you tested this on Windows with the emulator and got it working? I assume this also fixes https://github.com/microsoft/vscode-cosmosdb/issues/1763?

Reply:
Yes, Nathan and I tested this on his windows machine; it is working.